app/javascript/components/game/Game.js

```diff
@@ -1,30 +1,74 @@
 import React, { Component } from 'react'
-import './game.css'
+import GameForm from '../gameForm'
+import Cell from '../Cell'
+import '../game/game.css'
 
 class Game extends Component{
+    state = {
+        messages: {
+          board: '',
+          win:'',
+          draw:''
+        },
+        id: ''
+      }
+
+      addGame = (form) => {        
+        this.setState({
+            id: form.id
+          });
+        }
+
+        playGame = (output) => {  
+           
+          if (output&&output.message){
+          this.setState({
+              messages: output
+            });
+        }
+      }
+
     render(){
+        const { messages } = this.state
+        const cells = [];
+        for (let index = 0; index < 9; index++) { 
+          cells.push( <Cell 
+            key={index} 
+            id={this.state.id} 
+            playGame={this.playGame} 
+            position={index} 
+            message={messages.message && messages.message.board[index]}
+            endGame={messages.message && messages.message.win || messages.message && messages.message.draw }
+            />);
+        }
         return(
             <>
+            
                 <div className='game_left'>
-                    <div class="container">
-                        <h1>Tic-Tac-Toe</h1>
-                        <ul id="game">
-                            <li data-pos="0,0"></li>
-                            <li data-pos="0,1"></li>
-                            <li data-pos="0,2"></li>
-
-                            <li data-pos="1,0"></li>
-                            <li data-pos="1,1"></li>
-                            <li data-pos="1,2"></li>
-                            
-                            <li data-pos="2,0"></li>
-                            <li data-pos="2,1"></li> 
-                            <li data-pos="2,2"></li>
-                        </ul>
-                    </div>
-                </div>
-                <div className='game_right'>
-                </div>
+                  <h3>Current Game ID: {this.state.id}</h3>
+                  <div className="container">
+
+                    <div className="row">
+                      <div className="col-md-2">
+                        <GameForm addGame={this.addGame}/>
+                      </div>
+                      
+                      <div className="col-md-8">
+                          <h1>Tic-Tac-Toe</h1>
+                          <ul id="game">
+                            {cells}
+                          </ul>
+
+                        <div id="game-messages">
+                            <span className="game-over">{messages.message && messages.message.win}</span>
+                            <span className="game-over">{messages.message && messages.message.draw}</span>
```

Comment:
Considering that you can't have both win and draw at the same time, could we consolidate these two lines into one?